oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2TableInstaller.java

```diff
@@ -92,7 +92,7 @@ protected String getColumnType(ModelColumn column) {
         } else if (String.class.equals(type)) {
             return "VARCHAR(" + column.getLength() + ")";
         } else if (IntKeyLongValueHashMap.class.equals(type)) {
-            return "MEDIUMTEXT";
+            return "VARCHAR(20000)";
```

Comment:
Why change this?

Reply:
> Why change this?

<img width="669" alt="image" src="https://user-images.githubusercontent.com/15965696/77926626-e93fdb00-72d8-11ea-9b13-8b995672f956.png">

Reply:
But e2e still passed? 

Then you can't change this only. You need to override this method in the MySQL installer. Otherwise, the size of row could be over-length easily.

Reply:
> But e2e still passed?

https://github.com/apache/skywalking/issues/4594